src/slider/Slider.ts

```diff
@@ -109,10 +109,13 @@ export default class Slider extends SliderBase<SliderProperties> {
 			vertical = false,
 			verticalHeight = '200px'
 		} = this.properties;
-		const {
+		let {
 			value = min
 		} = this.properties;
 
+		value = value > max ? max : value;
+		value = value < min ? min : value;
```

Comment:
This isn't a suggestion so much as a question: my initial instinct is towards `Math.max(value, min)` and `Math.min(value, max)` for readability. Just curious if there's a rationale for one vs. the other apart from personal preference?